extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -270,6 +270,9 @@ export class SubscriptionService {
           'Selected service not registered');
 
       selectedPlatform.activate(renderState);
+      if (selectedPlatform.isPingbackEnabled()) {
+        selectedPlatform.pingback();
```

Comment:
We should also always pingback the local platform. And obviously, avoid pinging it back twice.

Reply:
Done